### PR TITLE
Fix/layout style

### PR DESCRIPTION
**Pull request by SohagHossain9983**
modify home page , card page checkout page

### CODE REVIEW

**Review by rahmanshifur — APPROVED** [review]
Good job

File: components/layout/header/index.jsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-import TopHeader from "./top-header";
+
+import BottomNavbar from "./bottom-navbar";
+
+import TopNavbar from "./top-navbar";
 
 const Header = () => {
     return (
-        <Fragment>
+        <div className="sticky-top">
```
Comment (naimuddin374) [inline components/layout/header/index.jsx:6]: fix coding issue

File: components/layout/header/top-navbar/header-action.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+import Link from 'next/link'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {faUser, faHeart, faCartShopping } from '@fortawesome/free-solid-svg-icons'
+
+
+
```
Comment (naimuddin374) [inline components/layout/header/top-navbar/header-action.jsx]: fix component name

File: styles/components/layout/bottom-navbar/index.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (naimuddin374) [inline styles/components/layout/bottom-navbar/index.scss:1]: fix file name should be _ before

File: styles/components/layout/bottom-navbar/modal-manu-item.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (naimuddin374) [inline styles/components/layout/bottom-navbar/modal-manu-item.scss]: fix file name

File: styles/components/order-summary/index.scss [hunk 5]
```diff
@@ -3,6 +3,9 @@
 #OrderSummary{
```
Comment (naimuddin374) [inline styles/components/order-summary/index.scss:3]: fix filename

File: styles/components/slider/index.scss [hunk 6]
```diff
@@ -11,7 +11,13 @@
         background: white;
```
Comment (naimuddin374) [inline styles/components/slider/index.scss:11]: fix filename

**Review by naimuddin374 — CHANGES_REQUESTED** [review]
follow the feedback

**Review by naimuddin374 — APPROVED** [review]

**Review by naimuddin374 — APPROVED** [review]

**Review by naimuddin374 — APPROVED** [review]

**Pull request merged**